unitTests/webfield-utils.test.js

```diff
@@ -523,6 +523,70 @@ describe('filterCollections', () => {
     )
     expect(result.filteredRows.map((p) => p.id)).toEqual([1, 3])
   })
+
+  test('filter profiles', () => {
+    const collections = [
+      {
+        id: 1,
+        metaReviewData: {
+          seniorAreaChairs: [
+            { preferredName: 'Name One', preferredEmail: 'one@email.com', type: 'profile' },
+          ],
+        },
+      },
+      {
+        id: 2,
+        metaReviewData: {
+          seniorAreaChairs: [
+            { preferredName: 'Name TWO', preferredEmail: 'two@email.com', type: 'profile' },
+          ],
+        },
+      },
+      {
+        id: 3,
+        metaReviewData: {
+          seniorAreaChairs: [
+            { preferredName: 'Name two', preferredEmail: 'three@email.com', type: 'profile' },
+          ],
+        },
+      },
+    ]
+    let filterString = 'sac=two'
+    const propertiesAllowed = {
+      sac: ['metaReviewData.seniorAreaChairs'],
+    }
+
+    let result = filterCollections(
+      collections,
+      filterString,
+      filterOperators,
+      propertiesAllowed,
+      uniqueIdentifier
+    )
+    expect(result.filteredRows.map((p) => p.id)).toEqual([2, 3])
+
+    // exact match
+    filterString = 'sac=="Name two"'
```

Comment:
Can we filter by tilde id too?

Reply:
yes, added in [98e23a1](https://github.com/openreview/openreview-web/pull/1830/commits/98e23a19285461925879021875bd46a4828576fc)